v2/tagging.md

```diff
@@ -461,6 +534,13 @@ Tag.Builder.create(key, value).build().visit(scope);
 new Tag(key, value).Visit(scope);
 ```
 
+------
+#### [ Go ]
+
+```
+awscdk.NewTag(key, value, &awscdk.TagProps{}).Visit(scope)
+```
+
 ------
 
 You can tag all constructs under a scope but let the values of the tags derive from properties of each construct\. To do so, write an aspect and apply the tag in the aspect's `visit()` method as shown in the preceding example\. Then, add the aspect to the desired scope using `Aspects.of(scope).add(aspect)`\.
```

Comment:
Can you add a code example for the final example, where docs has: "The following example applies a tag to each resource in a stack containing the resource's path."